pandas/_libs/tslibs/timedeltas.pyx

```diff
@@ -859,14 +859,6 @@ cdef class _Timedelta(timedelta):
         """
         return self.to_timedelta64()
 
-    def total_seconds(self):
```

Comment:
No longer need this implementation since we can rely on `timedelta.total_seconds`

Reply:
what change for the "no longer" part?

Reply:
Since we are now dispatching the correct microseconds to `datetime.timedelta` here: https://github.com/pandas-dev/pandas/pull/31380/files#diff-d23555c3fb6b3ffc9f2088e09f9ec5f9R1245

we can use `datetime.timedelta.total_seconds` as mentioned in this https://github.com/pandas-dev/pandas/pull/31155#discussion_r370721655 since we are no longer rounding.

Reply:
sounds good, thanks for explaining

Reply:
FWIW, I did consider suggesting the removal of `total_seconds` here, but when I benchmarked it, I found that the implementation where you truncate `._value` was about 2x as fast as the native `timedelta.total_seconds()`, if that matters at all.